components/x-teaser/src/concerns/image-service.js

```diff
@@ -8,7 +12,8 @@ const OPTIONS = ['source=next', 'fit=scale-down', 'dpr=2'];
  * @param {String} options
  */
 export default function imageService(url, width, options) {
-	const encoded = encodeURIComponent(url);
-	const href = `${BASE_URL}/${encoded}?${OPTIONS.join('&')}&width=${width}`;
-	return options ? href + '&' + options : href;
+  const imageSrc = new URL(`${BASE_URL}/${encodeURIComponent(url)}`);
```

Comment:
officially, `x-teaser` supports any Node version from 6; `URL` is only available on the global object from v10. i don't know if anything consuming `x-teaser` is on older Node versions, but so we don't have to remove support for those versions (which would be a breaking change), you should add `const {URL, URLSearchParams} = require('url')`

Reply:
Once the upgrade of all Customer Products apps to Node v12 is complete, and if all the App's repos likewise have a minimum of v10, we can then use the `URL` from the global object with confidence.